src/libostree/libostree.sym

```diff
@@ -373,12 +373,10 @@ global:
  *                         NOTE NOTE NOTE
  */
 
-/* Remove comment when first new symbol is added
-LIBOSTREE_2016.XX {
+LIBOSTREE_2017.3 {
```

Comment:
Uh, I don't think we want the symbol version to be 2017.3 on our version, do we? Our version is 2016.15.